File: WebApplication-Project/Camp Booking WebApp/WebUI/src/app/Models/Camp.ts
export interface Camp {
  id: string;
  name: string;
  description: string;
  capacity: number;
  rating: number;
  price: number;
  imageUrl: string;
}

export interface AddNewCamp {
  name: string;
  description: string;
  capacity: number;
  price: number;
  imageUrl: string;
}

export interface GiveRating {
  campId: string;
  referenceNumber: string;
  cellPhone: string;
  star: number;
  comment: string;
}

File: WebApplication-Project/Camp Booking WebApp/CampBooking/CampBooking.DAL/DbModel/Camp.cs
namespace CampBooking.DAL.DbModel
{
    public class Camp
    {
        public Guid Id { get; set; }
        public string Name { get; set; }
        public int Price { get; set; }
        public int Capacity { get; set; }
        public double Rating { get; set; }
        public string Description { get; set; }
        public string ImageUrl { get; set; }
    }
}
